style.css

```diff
@@ -1,20 +1,117 @@
-*{
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-    height: 100vh;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
-.box{
-    background-color: blueviolet;
-    border: 1px solid black;
-    width: 200px;
-    height: 50vh;
-    flex: 1;
+
+@media only screen and (max-width: 768px) {
+  body {
+    height: 100vh;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-color: #fff;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 4vh 5vh;
+  }
+
+  .top-logo h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size: 22px;
+    color: #6070ff;
+  }
+
+  .menu-icon {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    width: 30px;
+    height: 25px;
+  }
+
+  .menu-vector {
+    width: 30px;
+    height: 3px;
+    background-color: #6070ff;
+
+    /* border: 1px solid black; */
+  }
+
+  main {
+    display: flex;
+    background-color: rgb(243, 245, 247);
+    height: 100vh;
+    padding: 10vh 5vh;
+  }
```

Comment:
- [ ] Please use a background image for your main section shown in [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=upVfIj2GmLtoPwtT-0) as mentioned in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

> Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

You can export `images` and `icons` from `Figma`. Here is the [link](https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma) on how to export images from Figma.